lib/rubocop/cop/rspec/capybara/specific_matcher.rb

```diff
@@ -37,11 +38,6 @@ class SpecificMatcher < Base
             'select' => 'select',
             'input' => 'field'
           }.freeze
-          COMMON_OPTIONS = %w[
-            above below left_of right_of near count minimum maximum between text
-            id class style visible obscured exact exact_text normalize_ws match
-            wait filter_set focused
-          ].freeze
           SPECIFIC_MATCHER_OPTIONS = {
             'button' => (
               COMMON_OPTIONS + %w[disabled name value title type]
```

Comment:
I am not sure why, but I would prefer adding (part of) the namespace in front of the constant name, e.g.:
```suggestion
              CssSelector::COMMON_OPTIONS + %w[disabled name value title type]
```

Reply:
Indeed it is. I add namespace in front of the constant name.